pkg/sensors/exec/exec_test.go

```diff
@@ -1215,3 +1216,75 @@ func TestExecProcessCredentialsSetuidChanges(t *testing.T) {
 	err = jsonchecker.JsonTestCheck(t, checker)
 	assert.NoError(t, err)
 }
+
+// Detect execution of binaries with file capability sets
+func TestExecProcessCredentialsFileCapChanges(t *testing.T) {
+	var doneWG, readyWG sync.WaitGroup
+	defer doneWG.Wait()
+
+	ctx, cancel := context.WithTimeout(context.Background(), tus.Conf().CmdWaitTime)
+	defer cancel()
+
+	obs, err := observertesthelper.GetDefaultObserver(t, ctx, tus.Conf().TetragonLib, observertesthelper.WithMyPid())
+	if err != nil {
+		t.Fatalf("Failed to run observer: %s", err)
+	}
+
+	// The drop-privileges is a helper binary that drops privileges so we do not
+	// drop it inside this test which will break the test framework.
```

Comment:
The binary makes sense to me.
But don't we set the gid/uid elsewhere in the tests? Is that different?

Reply:
Ah the gid/uid for this case are set in the drop-privileges binaries to drop all privileges and caps, then it will execute ping binary to check for new gained caps